src/main/scala/com/github/witi83/akka/jwt/JwtDirectives.scala

```diff
@@ -226,28 +254,31 @@ object JwtClaimBuilder {
   /**
    * Returns a claim builder which sets the "exp" field to an expiration time.
    *
+   * If `duration` is less than one second, it will be treated as 0.
+   *
    * @param duration
-   * The valid duration of a JWT.
-   * The minimum resolution is one minute.
+   * Valid duration of a JWT.
+   * Minimum resolution is one second.
    */
   def claimExpiration[T](duration: Duration): SubjectExtrator[T] = input => {
-    val validUntil = new Date(Instant.now().plusSeconds(duration.toSeconds).toEpochMilli)
-    Some(new Builder().expirationTime(validUntil).build())
+    val validUntil = Calendar.getInstance()
```

Comment:
I'm new in Scala/Java world, in kikuomax repo Calendar was used, so I supposed to rewrite it. now I understand it was a mistake and it should be fixed (reverted back to Instant)